.github/RELEASE_SCHEDULE.md

```diff
@@ -1,16 +1,29 @@
 # DNN Platform Release Schedule
-To ensure adequate time for release planning by the community, partners, and vendors a specific release process will be followed for all releases.  
+To ensure adequate time for release planning by the community, partners, and vendors, a specific release process will be followed for all releases.  
 
 ## Release Candidates
-For a period of one week (Revision), two weeks (Minor) or three-weeks (Major) before any release, a Release Candidate (RC) version will be made available to the public.  At present these release candidates will be for testing only. After version 10.x efforts will be made to support upgrading from RC to Production releases.
+A Release Candidate (RC) is designed to give the community time to adjust their existing environments for any breaking changes and identify any unintended changes. Strong community participation during the RC process will result in more stable releases. 
 
-The goal of these release candidates is to give the community time to adjust their existing environments for any breaking changes, as well as to identify any issues with the changes.  If necessary, changes will be incorporated an additional RC release could be made if significant problems are identified.  If a revised Release Candidate is necessary the Production Release schedule may be impacted.  The exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during RC review, however, will be clearly communicated as part off the updated RC release notes.
+### Major Releases (`MAJOR.Minor.Patch`)
+Major releases will have an initial RC cycle with a minimum duration of three weeks between the date of the RC release and the date of the final release. 
 
-## Production Releases
-Production releases will only be completed after a successful RC phase, except in the case of a significant security release that was included as part of a revision release.
+### Minor Releases (`Major.MINOR.Patch`)
+Minor releases will have an initial RC cycle with a minimum duration of two weeks between the date of the RC release and the date of the final release.
+
+### Patch Releases (`Major.Minor.PATCH`)
+Patch releases will have an initial RC cycle with a minimum duration of one week from the RC release and final release.
+
+### Changes During RC Cycle
+If necessary, changes will be incorporated during an RC; if the changes resolve significant issues or introduce risk, an additional RC may be created at the Approvers group's discretion.  If a revised RC is necessary, the Final Release schedule will be impacted; the exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during the RC review. However, it will be communicated as part of the updated RC release notes.
```

Comment:
```suggestion
If necessary, changes will be incorporated during an RC; if the changes resolve significant issues or introduce risk, an additional RC may be created at the discretion of the Approvers group.  If a revised RC is necessary, the Production Release schedule will be impacted; the exact impact will vary on a case-by-case basis depending on the nature of the issue(s) identified during the RC review. However, it will be communicated as part of the updated RC release notes.
```